.buildkite/data.rayci.yml

```diff
@@ -93,6 +93,18 @@ steps:
     depends_on: datamongobuild
     job_env: forge
 
+  - label: ":database: data: dashboard tests"
+    tags:
+      - python
+      - data
+      - dashboard
+    instance_type: medium
+    commands:
+      - bazel run //ci/ray_ci:test_in_docker -- python/ray/dashboard/... data
```

Comment:
missing `--build-name datanbuild`

also datanbuild is a build with nightly pyarrow, there are also data6build and data12build which pins pyarrow to 6.* and 12.* version; not sure which one you really want just fyi